src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/TaskTransformer.kt

```diff
@@ -117,4 +119,9 @@ class TaskTransformer(
       userTransformer.transformJpaToApi(userEntity, ServiceName.approvedPremises) as ApprovedPremisesUser
     }
   }
+
+  // Use the sure operator here as entities will definitely have a `dueAt` value by the time they're surfaced as tasks
```

Comment:
presumably this has all been backfilled in prod/pre-prod before we merge this (or does the code ensure it's populated on retrieval?)

Reply:
Yeah, it's done in preprod, but I've yet to do it in prod. Will do that before merging this